source/guide_directus.rst

```diff
@@ -0,0 +1,163 @@
+.. highlight:: console
+
+.. author:: j3n57h0m45 <https://github.com/j3n57h0m45>
+
+
+.. tag:: self-hosting
+.. tag:: database
+
+.. sidebar:: About
+
+  .. image:: _static/images/directus.png
+        :align: center
+
+########
+Directus
+########
+
+.. tag_list::
+
+Directus_ is a real-time API and App dashboard for managing SQL database content.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`the shell <basics-shell>`
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here in the GitHub_ repository of the project:
+
+  * https://github.com/directus/directus/blob/main/license
+
+Prerequisites
+=============
+
+Node and npm
+------------
+
+We're using :manual:`Node.js <lang-nodejs>`, its package manager :manual_anchor:`npm <lang-nodejs.html#npm> as well as ``npx``:
```

Comment:
I Think you're missing a ` here. Check out the preview, the link is broken: https://deploy-preview-1063--uberlab.netlify.app/guide_directus.html

Reply:
fixed